transplant_guide.md

```diff
@@ -246,6 +251,7 @@ Get settled, girl! Consider planning an IKEA trip with your other cohort transpl
     The Seattle Public Library is an amazing public service (the central branch is 11 stories!) with many local branches that may be in your neighborhood. All you need is proof of a local mailing address (bring in a piece of mail or a copy of your lease) and you’ll have access to all the things!
 
   - **Change your driver's license**
-    What's the timeline?
+    By law, you have 30 days to change your driver's license and register your car with WA. But let's be honest, you're going to have a lot on your plate. So unless you happen to have a really nosy neighbor, I wouldn't stress it too much. Just make sure you get it done.
```

Comment:
I think it might be worth noting for anyone who's already a WA resident that the legal requirement is __10 days__. [source](http://www.dol.wa.gov/driverslicense/addresschange.html)